### PR TITLE
Feature/#3 #5: add mock data for cart and checkout

### PR DESCRIPTION
**Pull request by janwidmer**

### CODE REVIEW

File: projects/spartacus-mock/src/lib/defaultHandlers.ts [hunk 1]
```diff
@@ -75,6 +77,10 @@ export function getDefaultHandlers(environment: Environment): RestHandler[] {
       return res(ctx.status(200), ctx.json(countries()));
     }),
 
+    rest.get(routes.regions, (_req: RestRequest, res: ResponseComposition, ctx: RestContext) => {
+      return res(ctx.status(200), ctx.json(countries()));
```
Comment (claudio147) [inline projects/spartacus-mock/src/lib/defaultHandlers.ts:81]: Use the created method for regions()

File: projects/spartacus-mock/src/lib/mock-data/commerce/payment-sop.ts [hunk 2]
```diff
@@ -0,0 +1,271 @@
+import { getSharedAddress } from '../account/addresses';
+
+export const getPaymentSopRequest = (): object => {
```
Comment (claudio147) [inline projects/spartacus-mock/src/lib/mock-data/commerce/payment-sop.ts:3]: Do we need this as static as it is?

File: projects/spartacus-mock/src/lib/defaultHandlers.ts (hunk 1 above)
Comment (janwidmer) [reply projects/spartacus-mock/src/lib/defaultHandlers.ts:81]: done

File: projects/spartacus-mock/src/lib/mock-data/account/payments.ts [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+export const createPaymentDetails = (defaultPayment: boolean): object => {
   return {
-    accountHolderName: 'Hans Muster',
+    accountHolderName: `${faker.name.firstName()} ${faker.name.lastName()}`,
     billingAddress: getSharedAddress(false, false),
     cardNumber: '************1111',
     cardType: {
       code: 'visa',
       name: 'Visa',
     },
```
[the remark this answers is not in the document]
Comment (janwidmer) [reply projects/spartacus-mock/src/lib/mock-data/account/payments.ts:14]: done

File: projects/spartacus-mock/src/lib/mock-data/commerce/payment-sop.ts (hunk 2 above)
Comment (janwidmer) [reply projects/spartacus-mock/src/lib/mock-data/commerce/payment-sop.ts:3]: Made it a little bit more dynamic

**Pull request merged**